datumaro/cli/contexts/project/__init__.py

```diff
@@ -399,6 +402,52 @@ def filter_command(args):
 
     return 0
 
+def build_generate_parser(parser_ctor=argparse.ArgumentParser):
+    parser = parser_ctor(help="Generate synthetic dataset",
+        description="""
+        This command is useful for making a synthetic dataset
+        with specified shape and storing them in provided directory.|n
```

Comment:
Don't describe what it's useful for, describe what it _does_.

Reply:
done